server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestAddVotingConfigExclusionAction.java

```diff
@@ -36,7 +36,11 @@
     private static final TimeValue DEFAULT_TIMEOUT = TimeValue.timeValueSeconds(30L);
 
     public RestAddVotingConfigExclusionAction(RestController controller) {
+        // TODO This API is being deprecated.
         controller.registerHandler(RestRequest.Method.POST, "/_cluster/voting_config_exclusions/{node_name}", this);
+
+        controller.registerHandler(RestRequest.Method.POST,
+            "/_cluster/voting_config_exclusions/node_ids_or_names/{node_id_or_names}", this);
```

Comment:
I think you can register the bare URI `/_cluster/voting_config_exclusions` (no query parameters) and then consume the query parameters in code.

Reply:
Done.